src/exception/property-exceptions/util.ts

```diff
@@ -28,7 +28,9 @@ export function isProgramException(result: ProgramException | null): result is P
 export function isEntityException(result: EntityException | null): result is EntityException {
 	return (
 		result?.programId !== undefined &&
-		(result?.specimen !== undefined || result?.follow_up !== undefined)
+		(result?.specimen !== undefined ||
+			result?.follow_up !== undefined ||
+			result?.treatment !== undefined)
 	);
 }
```

Comment:
curious how well this was working before, if this change is happening now.
I guess `treatment` was not working with the `isEntityException` function

Reply:
No appears it was missing